packages/web-components/src/components/masthead/masthead-global-bar.ts

```diff
@@ -61,7 +62,11 @@ class C4DMastheadGlobalBar extends FocusMixin(
       [`${c4dPrefix}-ce--header__global__container--has-search-active`]:
         hasSearchActive,
     });
-    return html` <div class="${classes}"><slot></slot></div> `;
+    return html`
+      <div part="ce-header-global-container" class="${classes}">
```

Comment:
I think we can simplify this to just `container`.
```suggestion
      <div part="container" class="${classes}">
```